pyproject.toml

```diff
@@ -58,6 +58,9 @@ optional = true
 matplotlib = ">=3.4"
 memory-profiler = "^0.61.0"
 
+[tool.poetry.scripts]
+spp-convert = "space_packet_parser.converters:main"
```

Comment:
If we have a different command line utility named `spp`, do we want this to be a subcommand like `spp convert /path/to/file` instead of a totally separate CLI?